geode-core/src/main/java/org/apache/geode/cache/RegionFactory.java

```diff
@@ -769,8 +784,7 @@ public void setPublisher(boolean v) {}
    * @throws CacheClosedException if the cache is closed
    * @since GemFire 7.0
    */
-  @SuppressWarnings("unchecked")
-  public Region<K, V> createSubregion(Region<?, ?> parent, String name)
+  public Region<K, V> createSubregion(Region<K, V> parent, String name)
```

Comment:
Fixed typing warning.

Reply:
Is this a breaking API change?

Reply:
Not that I can see. All relevant testing still passes. It was getting a warning that the type was not correct, hence the reason I changed it to clear the unchecked.

Reply:
I think this forces the parent and child to have the same K,V types?
 Cache cache = getCache();
      RegionFactory<String, String> regionFactory = cache.createRegionFactory(RegionShortcut.REPLICATE);
      Region<String, Integer> parentRegion = cache.getRegion(PARENT_REGION_NAME);
      regionFactory.createSubregion(parentRegion, CHILD_REGION_NAME);

I believe that would have worked before but now it will show as an issue since the parent and child regions have different types (which actually is allowed in geode)

Reply:
Does this force the parent and child to have the same K,V types?  If so, I don't think that is a valid requirement for Geode...